src/Whoops/Run.php

```diff
@@ -395,6 +410,13 @@ private function writeToOutputNow($output)
         return $this;
     }
 
+    private static function isInternalErrorException($exception)
+    {
+        return class_exists('Error') &&
+            !is_subclass_of('Error', 'Exception') &&
```

Comment:
Yeah, exactly.
I suppose a PHP version check should be good enough here, but I got used to the idea of feature-checking instead of version-checking from other environments.
In particular, I have no idea how HHVM reports it's version number and which features it implements.